multiqc/utils/config_defaults.yaml

```diff
@@ -169,6 +169,7 @@ module_order:
     - 'rseqc'
     - 'busco'
     - 'goleft_indexcov'
+    - 'disambiguate'
```

Comment:
I'd class this as _'Pre-alignment QC'_ myself I think - it probably fits best alongside `fastq_screen` below, the two tools do similar tasks.